### PR TITLE
PRIPAD-5152 Display list of apps in table when on desktop

### PR DESCRIPTION
**Pull request by remover**

### CODE REVIEW

File: server/php/includes/assetdirectory.php [hunk 1]
```diff
@@ -93,33 +92,28 @@ public function getApplicationVersions($platform) {
...
                         $version[AppUpdater::FILE_ANDROID_APK] = $subAssetDir->apk;
                         $version[AppUpdater::FILE_ANDROID_JSON] = $subAssetDir->json;
                         $version[AppUpdater::FILE_COMMON_NOTES] = $subAssetDir->note;
-                        $allVersions[$subDir] = $version;
                         
-                        if ($icon == null) {
-                            $icon = $subAssetDir->icon;
-                        }
+
+                        $version[AppUpdater::FILE_COMMON_ICON] = $subAssetDir->icon;;
```
Comment (sebskuse) [inline server/php/includes/assetdirectory.php]: Double colon here

File: server/php/includes/assetdirectory.php [hunk 2]
```diff
@@ -128,17 +122,17 @@ public function getApplicationVersions($platform) {
                 $version[AppUpdater::FILE_IOS_IPA] = $this->ipa;
                 $version[AppUpdater::FILE_IOS_PLIST] = $this->plist;
                 $version[AppUpdater::FILE_COMMON_NOTES] = $this->note;
+                $version[AppUpdater::FILE_COMMON_ICON] = $this->icon;;
```
Comment (sebskuse) [inline server/php/includes/assetdirectory.php]: Ditto these here

**Comment by sebskuse** [comment]
Looks nice - good job

**Comment by remover** [comment]
Updated

**Pull request merged**